### PR TITLE
Update docs 6

### PR DESCRIPTION
**Pull request by vilem-k**
Updates by [https://3.basecamp.com/5435868/buckets/29206288/todos/6395237505](https://3.basecamp.com/5435868/buckets/29206288/todos/6395237505)

### CODE REVIEW

File: docs/pages/2_admin/02_users.md [hunk 1]
```diff
@@ -15,34 +15,45 @@ Users in a Shieldoo Secure Network are clients that can connect to servers.
...
 {: .tip }
 > - Use the __Invited__ button to only display users who have yet to accept their invitations.  
 > ![](../../images/Users02.png)
 > - Use the provided search field for specific users by name or email address.  
 > ![](../../images/Users03.png)
 
+{: .note }
+>If you invite the users in Admin | Expert mode:
+>- You can use the _Single user_ or _Multiple users_ option.
+>- You must select a specific Available template card. For more information, see [Access Card Templates](/access_management/#access-card-templates).
```
Comment (pavel-john) [inline docs/pages/2_admin/02_users.md:45]:
```suggestion
>- You have to select a specific Available template card. For more information, see [Access Card Templates](/access_management/#access-card-templates). 
```

File: docs/pages/2_admin/03_servers.md [hunk 2]
```diff
@@ -11,82 +11,185 @@ Servers in a Shieldoo Secure Network are devices to which users or other servers
...
-2. Use the provided options to edit the server:  
-   - __Detail__ - Open the installation instructions you used to install the server.
+   {% include img.html src="../../images/Servers02.gif" alt="Editing Servers" %}
+
+3. Use the provided options to edit the server:  
+   - __Detail__ - Open the current information about the server and the installation instructions you used to install the server.
    - __Edit__ - Open the server's configuration, which you can edit.
-   - __Delete__ - Deletes the server from your Shieldoo Secure Network.  
-   No users or other servers will be able to connect to it.
+   - __Delete__ - Deletes the server from your Shieldoo Secure Network. No users or other servers will be able to connect to it.
```
Comment (pavel-john) [inline docs/pages/2_admin/03_servers.md:63]: An idea to add: "... and its configuration will be forever lost."

File: docs/pages/2_admin/03_servers.md [hunk 3]
```diff
@@ -11,82 +11,185 @@ Servers in a Shieldoo Secure Network are devices to which users or other servers
...
+<!-->
+ADD TYPE OF CONNECTIONS: PEER-TO-PEER, RELAY, WEBSOCKET
+- Diagram image
+- Explanation
+- Troubleshooting
+<-->
+
+The icons in the list of servers in the __Network__ \| __Servers__ section provide visual information about the current server status:
+- ![](../../images/Servers03.png)
+The server was never connected to the network, and other enabled nodes (users and servers) can not connect to this server. The server may not have been installed yet.
```
Comment (pavel-john) [inline docs/pages/2_admin/03_servers.md:82]:
```suggestion
The server has never been connected to the network, and other enabled nodes (users and servers) cannot connect to this server. The server may not have been installed yet.
```

File: docs/pages/2_admin/04_access_management.md [hunk 4]
```diff
@@ -139,13 +139,16 @@ The following steps describe how to create access cards for servers:
...
    - __Firewall configuration__ - Select one of the prepared firewalls.
    - __Groups__ - Map the server with all the desired groups.
-   - __Punch back__ - Enable this check box if you want the node that you are trying to reach to connect back to your if your <a href="https://en.wikipedia.org/wiki/UDP_hole_punching" target="blank">UDP hole punching<a/> fails. This is useful if a node is behind, for example, a symmetric NAT.
-   - __Use websocket gateway__ - Check this check box if you want to use the Shieldoo websocket gateway to enforce the connection even from a site where, for example, UDP is not allowed.
    - __Valid Till__ - Define how long the access card will remain valid:
      - _Subscription lifetime_ - Defines that the access card will remain valid if your organisation subscribes to Shieldoo.
      - _Custom expiration date_ - Defines a custom expiration using the provided date picker.
 8. Click the __Save__ button to confirm the access card's creation.
 
+<!-->
```
Comment (pavel-john) [inline docs/pages/2_admin/04_access_management.md:147]: Forgotten comment?

File: docs/pages/2_admin/04_access_management.md [hunk 5]
```diff
@@ -139,13 +139,16 @@ The following steps describe how to create access cards for servers:
...
+   - __Punch back__ - Enable this check box if you want the node that you are trying to reach to connect back to your if your <a href="https://en.wikipedia.org/wiki/UDP_hole_punching" target="blank">UDP hole punching<a/> fails. This is useful if a node is behind, for example, a symmetric NAT.
```
Comment (pavel-john) [inline docs/pages/2_admin/04_access_management.md:148]:
```suggestion
   - __Punch back__ - Enable this check box if you want the node that you are trying to reach to connect back to you when your <a href="https://en.wikipedia.org/wiki/UDP_hole_punching" target="blank">UDP hole punching<a/> fails. This is useful if a node is behind a symmetric NAT, for example.
```

File: docs/pages/2_admin/04_access_management.md [hunk 6]
```diff
@@ -139,13 +139,16 @@ The following steps describe how to create access cards for servers:
...
+   - __Use websocket gateway__ - Check this check box if you want to use the Shieldoo websocket gateway to enforce the connection even from a site where, for example, UDP is not allowed.
```
Comment (pavel-john) [inline docs/pages/2_admin/04_access_management.md:149]:
```suggestion
   - __Use websocket gateway__ - Check this check box if you want to use the Shieldoo websocket gateway to establish a connection even from a site with restricted network conditions where UDP is not allowed, for example.
```

**Review by pavel-john — APPROVED** [review]
Some suggestions. Please validate my ideas ;)

File: docs/pages/2_admin/04_access_management.md (hunk 4 above)
Comment (vilem-k) [reply docs/pages/2_admin/04_access_management.md:147]: Left for a future generation. 😉

**Pull request merged**